src/r502_driver_utils.c

```diff
@@ -138,6 +141,9 @@ uint32_t get_command_reply_len(CommandType type) {
     case Store:
         len = STORE_REPLY_LEN;
         break;
+    case AuraLedConfig:
```

Comment:
Keep the same order of the commands as in the `cmd_func` array.

Reply:
Done!